src/basic/index.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * Basic Auth plugin.
+ */
+class BasicAuth {
+  constructor({ loginPath = '/login', logoutPath = '/logout', fetchUser = null } = {}) {
```

Comment:
```suggestion
  constructor({ loginPath , logoutPath , fetchUser } = { loginPath: '/login', logoutPath: '/logout' }) {
```

No need to have default value for `fetchUser` since you require it below.